src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/RenameDriver.cs

```diff
@@ -59,6 +59,15 @@ public string ApplyRename(string expressionText)
             return ExpressionLocalizationHelper.ConvertExpression(converted, strippedRenameParameters, BindingConfig.Default, _resolver, _binderGlue, CultureInfo.InvariantCulture, Features.None, false);
         }
 
+        public string RenameOptionSet(string expressionText)
```

Comment:
Why do we need a separate RenameOptionSet method? If you've set the _renameResolver/_renameParameters right in the constructor, wouldn't ApplyRename work?

Reply:
Correct. Let me remove it